tools/reviewer/web/src/app/dashboard/diff/diff-discussion/diff-discussion.component.html

```diff
@@ -0,0 +1,53 @@
+<!-- Title -->
+<div class="cr-titlebar">
+  <div class="title">
+    <span class="bold">Discussion</span>
+  </div>
+  <div class="buttons" fxFlex>
+    <button
+      class="cr-light-button"
+      mat-raised-button
+      color="primary"
+    >Expand</button>
+  </div>
+</div>
+<div class="cr-header">
+  <mat-table [dataSource]="threads">
+      <ng-container matColumnDef="discussions">
+        <mat-header-cell *matHeaderCellDef>
+          General discussions
+          ({{getAmountOfComments()}} comments,
+          {{getAmountOfComments()}} unresolved)
+        </mat-header-cell>
+        <!-- Discussions -->
+        <mat-cell *matCellDef="let thread">
+          <div class="thread-title">
+            {{ thread.getFile().getFilename() }}
+            ({{ thread.getCommentList().length }} comments)
+          </div>
+          <div
+            class="comment"
+            *ngFor="let comment of thread.getCommentList()"
+          >
+            <div class="username">{{ comment.getContent() }}</div>
+            <div class="message">
+              by <span class="bold">{{ getUsername(comment) }}</span>
```

Comment:
Also, something worth discussing. @oferb should there be classes in HTML that don't have any styles attached to them. They do help in identifying what functionality is encapsulated in the HTML element but they could also create too many class names.

Reply:
Readability of code is very important. What's the drawback in having many class names?

Reply:
having many classes might clutter the code. That's all I could think of.

Reply:
I don't feel strongly about this. The guiding principle is that the code should be fun to work with, for the author and for others.